### PR TITLE
[send] Added timvisee Send chart

### PR DESCRIPTION
**Pull request by geraldwuhoo**
Signed-off-by: geraldwuhoo <gerald@geraldwu.com>

**Description of the change**

Add [Send](https://gitlab.com/timvisee/send) chart

**Benefits**

People will be able to deploy Send, a simple drag-and-drop file sharing service with client-side encryption.

**Possible drawbacks**

NA

**Applicable issues**

NA

**Additional information**

**Checklist** 
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Chart `artifacthub.io/changes` changelog annotation has been updated in `Chart.yaml`. See [Artifact Hub documentation](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations) for more info.
- [x] Variables have been documented in the `values.yaml` file.

### CODE REVIEW

File: charts/stable/send/values.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: registry.gitlab.com/timvisee/send
+  # -- image tag
+  # @default -- chart.appVersion
+  tag: v3.4.20
```
Comment (bjw-s) [inline charts/stable/send/values.yaml:13]:
```suggestion
  tag:
```

**Review by bjw-s — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a tiny change requested :)

**Comment by geraldwuhoo** [comment]
Thanks! For future reference, why not include the tag in the base `values.yaml`?

File: charts/stable/send/values.yaml (hunk 1 above)
Comment (geraldwuhoo) [reply charts/stable/send/values.yaml]: Done

**Comment by bjw-s** [comment]
Thank you for updating! 

> Thanks! For future reference, why not include the tag in the base `values.yaml`?

The reason for this is that the value gets populated by the `appVersion` field from `Chart.yaml` metadata by default when the field is empty. We had quite some charts in the past where the `appVersion` metadata would differ from the `image.tag` value. So by leaving it empty by default we ensure that they are always in sync (and only have to update the application version in one spot) :) Users can still manually override/pin the value by specifying a value themselves.

**Review by bjw-s — APPROVED** [review]

**Comment by bjw-s** [comment]
@all-contributors Please add @geraldwuhoo for code

**Pull request merged**